docs/user-guides/observability.md

```diff
@@ -1407,6 +1407,24 @@ The examples below are all with `--log-level=debug` and `--log-mode=production`.
 
 ## Tracing
 
-Most log messages associated with an authorization request include a `request id` value. This value represents the ID of the external authorization request received and processed by Authorino and it is useful for matching incomming request and corresponding outgoing response log messages, as well as the more fine-grained log details available only in `debug` level.
+### Request ID
 
-In the future, Authorino will also emit telemetry that will help improve tracing ([#163](https://github.com/Kuadrant/authorino/issues/163)), as well as provide the ability to send logs of the authorization response to external services as a per-AuthConfig setting ([#364](https://github.com/Kuadrant/authorino/issues/364)).
+Processes related to the authorization request are identified and linked together by a _request ID_. The request ID can be:
+* generated outside of Authorino and passed in the authorization request – this is essentially the case of requests via GRPC authorization interface initiated by the Envoy;
+* generated by Authorino – requests via [Raw HTTP Authorization interface](../architecture.md#raw-http-authorization-interface).
+
+### Propagation
+
+Authorino propagates trace identifiers compatible with the W3C Trace Context format (https://www.w3.org/TR/trace-context/) and user-defined baggage data in the W3C Baggage format (https://www.w3.org/TR/baggage).
+
+### Log tracing
+
+Most log messages associated with an authorization request include the [`request id`](#request-id) value. This value can be used to match incomming request and corresponding outgoing response log messages, including at deep level when more fine-grained log details are enabled ([`debug` level level](#log-levels-and-log-modes)).
+
+### OpenTelemetry integration
+
+Integration with an OpenTelemetry collector can be enabled by supplying the `--tracing-service-endpoint` command-line flag (e.g. `authorino server --observability-service-endpoint=http://jaeger:14268/api/traces`).
+
+The additional `--tracing-service-tags` command-line flag allow to specify fixed agent-level key-value tags for the trace signals emitted by Authorino (`key1=value1[,keyN=valueN]*`).
```

Comment:
have you considered supporting multiple flags? That would leave the parsing logic to the CLI lib.

```
authorino server --tracing-service-tags a=1 --tracing-service-tags b=2
```

Reply:
I have not, but it's a good idea. It feels more Golang flags.

Done.